examples/resources/generate.go

```diff
@@ -24,6 +24,7 @@
 //go:generate file2byteslice -package=images -input=./images/smoke.png -output=./images/smoke.go -var=Smoke_png
 //go:generate file2byteslice -package=images -input=./images/tile.png -output=./images/tile.go -var=Tile_png
```

Comment:
Optional: you can move this so that the list is in alphabetical order

Reply:
It looks like there are other entries already not in alphabetical order... 🤷

Reply:
Ack